framework/src/main/java/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -490,6 +494,7 @@ private void addFinalLocalValues(S store, Element enclosingElement) {
      * @param methodDeclTree the declaration of the method or constructor
      * @return true if the receiver of a method or constructor might not yet be fully initialized
      */
+    @SideEffectFree
```

Comment:
Could this be Pure?

Reply:
SideEffectFree, Deterministic, Pure annotation are trusted annotations.